tests/test.py

```diff
@@ -4253,6 +4255,7 @@ def testStartupWithoutVersionInConfig(self):
             default_args = {
                 'acra-server': ['-db_host=127.0.0.1'],
                 'acra-connector': ['-user_check_disable', '-acraserver_connection_host=127.0.0.1', '-client_id=keypair1'],
+                'acra-migrate-keys': ['--dry_run', '--src_keydir={}'.format(KEYS_FOLDER.name)],
```

Comment:
it's cool that you found these place in our 4.4k lines file with tests where you can test starting tool ))

Reply:
Well, no silver bullet here, I just followed the stack trace:

    Traceback (most recent call last):
      File "tests/test.py", line 4282, in testStartupWithOutdatedConfigVersion
        stderr = self.getOutputFromProcess(args)
      File "tests/test.py", line 1224, in getOutputFromProcess
        process = subprocess.Popen(args, stderr=subprocess.PIPE)
      File "/usr/lib/python3.5/subprocess.py", line 947, in __init__
        restore_signals, start_new_session)
      File "/usr/lib/python3.5/subprocess.py", line 1551, in _execute_child
        raise child_exception_type(errno_num, err_msg)
    FileNotFoundError: [Errno 2] No such file or directory: './acra-migrate-keys'

If not for that, I'd still be confused about what has gone wrong. So, ugh, thanks for using Python, I guess :)